docs/getting-started/README.md

```diff
@@ -72,3 +72,7 @@ Start the site with `npm run dev`. The app will then watch for changes in client
 ::: tip
 If you are starting the site in a production or do not want the process to watch for changes, start the site with `node app.js`.
 :::
+
+## Next steps
+
+Now that Apostrophe is installed, you're ready to start building. Check out the essentials [guide](/guide/) to get started or read more about Apostrophe's inner workings in the [technical overview](/getting-started/technical-overview.md).
```

Comment:
inner workings still seems like the wrong phrase here. What are we really putting in this technical overview? Should we just...

```suggestion
Now that Apostrophe is installed, you're ready to start building. Check out the essentials [guide](/guide/) to get started or read the [technical overview](/getting-started/technical-overview.md) for an overview of Apostrophe's structure.
```

Reply:
Why is it not? It's colorful language, but [not slang](https://www.merriam-webster.com/dictionary/inner%20workings). And it's general enough to encompass both the directory structure and the tech stack stuff, the latter of which is not included in "Apostrophe's structure."